ui/admin/app/templates/application.hbs

```diff
@@ -132,33 +132,43 @@
 {{/if}}
 
 <PendingConfirmations as |confirmation accept deny|>
-  <Rose::Dialog
-    @heading={{if
-      confirmation.options.title
-      (t confirmation.options.title)
-      (t 'actions.confirm')
-    }}
-    @dismissButtonText={{t 'actions.cancel'}}
-    @icon='flight-icons/svg/alert-triangle-16'
-    @style='warning'
-    @modal={{true}}
-    @dismiss={{deny}}
-    as |dialog|
-  >
-    <dialog.body>
-      <p>{{confirmation.text}}</p>
-    </dialog.body>
-    <dialog.footer>
-      <Rose::Button @style='primary' {{on 'click' accept}}>
-        {{if
-          confirmation.options.confirm
-          (t confirmation.options.confirm)
-          (t 'actions.ok')
-        }}
-      </Rose::Button>
-      <Rose::Button @style='secondary' {{on 'click' deny}}>
-        {{t 'actions.cancel'}}
-      </Rose::Button>
-    </dialog.footer>
-  </Rose::Dialog>
+  <Hds::Modal @color='warning' as |M|>
```

Comment:
For the alert dialog we can assume it will be just one dialog element on the DOM at a time, so I think we can use the standard selector `dialog`. I will suggest to not use `data-test-*` selector for this dialog use case.

But in case we wanted to unique identify this dialog, I will add the attribute to the `<HDS::Modal>` element above, since its the parent of the component.

Reply:
@calcaide Thanks for the feedback. I am wondering why we would continue using `dialog` as a test selector on an Hds::Modal component? Esp since we are retiring `Rose::Dialog` from the codebase.